src/components/Home/Home.tsx

```diff
@@ -1,35 +1,66 @@
 import React, { useState, useEffect } from 'react'
+import { fetchGetResponse } from '../../utils/apiCalls'
+import { cleanMarketsData, cleanDetailsData } from '../../utils/utils'
 
 interface ApiMarkets {
   markets: {
     id: number,
+    distanceFromZip: number,
     marketName: string
-    //need logic to split the distance from the name to display the name
   }[],
   marketDetails: {
-    googlink: string,
-    address: string,
-    schedule: string,
-    products: string
-    // need logic for splitting products string into an array
+    street: string,
+    city: string,
+    state: string,
+    zip: string,
+    schedule: {
+      dayOfWeek: string,
+      time: string,
+      season: string
```

Comment:
What does season look like?

Reply:
typically ~"11/07/2020 to 11/07/2020", but there's a conditional to make it "bad format" since I found this in one of the response objects "11/07/2020 to "